util/src/main/scala/com/scalableminds/util/tools/ConfigReader.scala

```diff
@@ -1,28 +1,10 @@
 package com.scalableminds.util.tools
 
-import java.time.Duration
-
-import play.api.Configuration
+import play.api.{ConfigLoader, Configuration}
 
 trait ConfigReader {
   def raw: Configuration
 
-  def getString(path: String): String =
-    openOrThrowException(path, raw.getString(path), "String")
-
-  def getInt(path: String): Int =
-    openOrThrowException(path, raw.getInt(path), "Int")
-
-  def getBoolean(path: String): Boolean =
-    openOrThrowException(path, raw.getBoolean(path), "Boolean")
-
-  def getDuration(path: String): Duration =
-    openOrThrowException(path, Some(raw.underlying.getDuration(path)), "Duration")
-
-  def openOrThrowException[T](path: String, value: Option[T], typ: String) = {
-    value match {
-      case Some(value) => value
-      case None => throw new Exception(s"Required config value is missing: $path (type $typ)")
-    }
-  }
+  def get[A](path: String)(implicit loader: ConfigLoader[A]): A =
+    raw.get[A](path)
```

Comment:
This was actually changed now in Play, the defaults are to be in the application.conf, no longer in code. (Just as we did already). It still makes sense for us to have the ConfigReaders, though, because they fail early, rather than when accessed.